src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: container-width($square-size, 7, $sides-paddings);
+  padding: 10px;
+
+  @each $day, $day-value in $week-days {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: ($square-size + $square-gap) * $day-value;
+    }
+  }
+
+  @for $amount from 28 through 31 {
+    &--month-length-#{$amount}
+    &__day:nth-child(n + #{$amount + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use `*` selector for it in normalize